<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupEarthCoverage</class>
 <widget class="QWizard" name="SetupEarthCoverage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>395</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>272</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Define earth coverage areas and lines</string>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ClassicStyle</enum>
  </property>
  <property name="options">
   <set>QWizard::HelpButtonOnRight</set>
  </property>
  <widget class="QWizardPage" name="wizardPage_2">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>2- Choose the frame layer</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>1- Enter Earth Coverage Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="nameLineEdit"/>
    </item>
    <item row="3" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="filterLineEdit"/>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget"/>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::MinimumExpanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage1">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>3- Choose the area classes that build earth coverage (only parent classes are listed here).</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="CustomSelector" name="areasCustomSelector" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>4- Choose the line classes that, along with the frame, build the areas of earth coverage 
(only parent classes are listed here).</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="CustomSelector" name="linesCustomSelector" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>5- For each area class of earth coverage, define which lines are used to build it 
(note that frame is always used to close areas)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="itemsExpandable">
       <bool>true</bool>
      </property>
      <attribute name="headerCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="headerDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="headerMinimumSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="headerStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Class</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Delimiters</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomSelector</class>
   <extends>QWidget</extends>
   <header>DsgTools.gui.CustomWidgets.SelectionWidgets.customSelector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
